test/topology_custom/test_repair.py

```diff
@@ -259,3 +259,10 @@ async def test_repair_abort(manager):
     await manager.api.client.get_json(f"/task_manager/wait_task/{id}", host=servers[0].ip_addr)
     statuses = await manager.api.client.get_json(f"/task_manager/task_status_recursive/{id}", host=servers[0].ip_addr)
     assert all([status["state"] == "failed" for status in statuses])
+
+@pytest.mark.asyncio
+@skip_mode('release', 'error injections are not supported in release mode')
+async def test_keyspace_drop_during_data_sync_repair(manager):
+    [await manager.server_add(config={
+        'error_injections_at_startup': ['get_keyspace_erms_throw_no_such_keyspace']
+    }) for _ in range(2)]
```

Comment:
How does that test repair? Due to the various system distributed keyspaces that still exist?

Eventually they'll be moved to tablets, and then repair won't be called as part of bootstrap. The test will then stop testing anything.